polaris-react/src/components/Image/Image.tsx

```diff
@@ -16,23 +21,44 @@ export interface ImageProps extends React.HTMLProps<HTMLImageElement> {
   onError?(): void;
 }
 
-export function Image({sourceSet, source, crossOrigin, ...rest}: ImageProps) {
+export function Image({
+  alt,
```

Comment:
How was this missing 🤦

Reply:
It wasn't actually I'm just exposing it. It was a required prop, just bundled up in `...rest`. Exposing it means we don't have to disable the a11y lint rule

Reply:
I see. I like getting it out of the rest 💯